core/templates/domain/statistics/LearnerActionObjectFactorySpec.ts

```diff
@@ -208,9 +208,9 @@ describe('Learner Action Object Factory', () => {
     };
 
     expect(() => {
-      // TS ignore is used because playthrough dict is assigned a invalid type
-      // to test errors.
-      // @ts-ignore
+      // Here playthrough dict is assigned a invalid action_type in order to
```

Comment:
Here, the playthrough dict has been assigned ...

(not "is", the assignment isn't happening in the line below. Also missing "the".)

Reply:
done